addons/craft/create_recipes.py

```diff
@@ -118,10 +121,10 @@ def get_recipes_from_soup(soup, spheres=False):
 
 def get_items_dictionary():
     if platform.system() == 'Windows':
-        path = 'C:\\Program Files (x86)\\Windower4\\res\\items.lua'
+        path = 'C:/Program Files (x86)/Windower/res/items.lua'
```

Comment:
This really bothers me, but it bothered me before your change as well. Why is this hard coded instead of using a relative path, or alternatively a path passed in via the command line (or both, with the relative path being a fallback)?

Also, backwards slashes are correcterer.

Reply:
The reason is mostly laziness.  The python scripts are things that a developer or user might periodically run to update the recipes and create a pull request, so much less thought was put into these kings of details.

Reply:
Considering most people will run this script from their Windower installation I'd just make it relative. This path would break for me, for example, as well as for many people who followed the common advice we gave of where to install Windower to until recently. That falls well within the limits of laziness imo :)

Ideally we'd have another mechanism for specifying the path, but those are obviously more work and would exceed the purpose of the PR, especially since this is nothing the PR changed.